### PR TITLE
i116: Seed Lewis Lockers

### LINKED ISSUE #116: Add new lewis lockers in bulk
### User story

As a lockers administrator at Lewis, I would like the lockers to be loaded in this system before implementing it, so that I don't have to manually create each locker by hand. This is a one time task to see the system done by a developer possibly as a seed file and likely done with a rack task. 

### Implementation notes, if any

There are 104 lockers at Lewis (52 on each floor).  They are on the 3rd and 4th floors, and have the floor number as the first digit in their number. First lockers are 301 and 401. 

- Start with populating only locker number (in "location" field) and floor
- May need to update database validation to allow for lockers with less information

### PR DESCRIPTION
**Pull request by sandbergja**
Adds a rake task `lockers:lewis:seed` to add these lockers.  It doesn't run by default; we'll have to run it manually.

closes #116

### CODE REVIEW

**Review by kevinreiss — APPROVED** [review]

File: spec/lib/tasks/lockers_rake_spec.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
```
Comment (maxkadel) [inline spec/lib/tasks/lockers_rake_spec.rb:1]: I haven't used this kind of test before for rake tasks, this is great!

File: spec/lib/tasks/lockers_rake_spec.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+      expect(Locker.find_by(location: '316').general_area).to eq('3rd floor')
+      expect(Locker.find_by(location: '435').general_area).to eq('4th floor')
+    end
+
+    it 'does not add a Lewis locker if it already exists' do
+      Locker.create! location: '401', general_area: '4th floor', building: Building.find_or_create_by(name: 'Lewis Library')
+      expect do
+        run_described_task
+      end.to change(Locker, :count).by(103)
+    end
```
Comment (maxkadel) [inline spec/lib/tasks/lockers_rake_spec.rb:38]: Is there a way to add an idempotence test? Maybe something like 

```suggestion
    end

    before do
      run_described_task
    end
    
    it 'is idempotent' do
      expect do
        run_described_task
      end.not_to change(Locker, :count)
    end
```

**Review by maxkadel — COMMENTED** [review]
Looks good!

A question to consider - If our Lewis locker applications are going to record a preferred size, should we give the lockers a size, even though it's the same?

**Review by maxkadel — APPROVED** [review]
Looks great, just that one failing test, which hopefully is an easy fix!

**Pull request merged**